### PR TITLE
clientupdate: clarify how to run update as Administrator on Windows

### PR DESCRIPTION
**Pull request by awly**
Make the error message a bit more helpful for users.

Updates #9456

### CODE REVIEW

File: clientupdate/clientupdate.go [hunk 1]
```diff
@@ -717,7 +717,7 @@ func (up *Updater) updateWindows() error {
 	}
 
 	if !winutil.IsCurrentProcessElevated() {
-		return errors.New("must be run as Administrator")
+		return errors.New("update must be run as Administrator; right-click cmd.exe, select 'Run as administrator' and try again")
```
Comment (noncombatant) [inline clientupdate/clientupdate.go:720]: Well, this requires the person to find cmd.exe in Explorer, which is a lot of clicking (even if they know the path to it). I suggest perhaps:

* Windows+x then a
* Windows+r then type "cmd" then Control+Shift+Enter

There is also `runas /user:Administrator "tailscale update"`, but that requires you to know the Administrator password — and on the default set up of at least Windows 11, there isn't one. So `runas` would not work for everyone. The other 2 methods above I think do.

Comment (awly) [reply clientupdate/clientupdate.go:720]: If the user ended up running `tailscale.exe update` in non-admin console of some kind (that's how they ended up in this code path), they can probably find `cmd.exe` again?
On Windows 11 you don't even need to find it in Explorer, you can use the search bar and right click from there.

I can swap the message to mention shortcuts, if you think that's easier for most users though.

Comment (noncombatant) [reply clientupdate/clientupdate.go:720]: Well, by default, tailscale.exe's parent directory is added to the `PATH` environment variable, so it is possible to find yourself running tailscale from the CLI without necessarily knowing where it is. It seems most likely that they ended up in `cmd` from Windows+r.

Maybe something like, "You can run cmd as Administrator by: ..." and then list all 3 possibilities? Up to you.

**Review by noncombatant — APPROVED** [review]

File: clientupdate/clientupdate.go (hunk 1 above)
Comment (awly) [reply clientupdate/clientupdate.go]: Sounds good, turns out multiline error string gets printed nicely!

**Pull request merged**